### DISCUSSION TITLE
Github Student Packages

### DISCUSSION THREAD

**Discussion by ammarkhattab** [community/community#28117] [Actions]
hello i am having a problem when i apply to student package is that it tells me that my university doesn't exist in my region while i have an id and really go to university in my region so what can i do

**Comment by atuschek0** [comment]
I had a similar issue although it was because GitHub didn't accept my school's email domain. Which is odd because my school was even listed in the list of schools GitHub provided. I also had trouble because my student ID because it doesn't have an expiry. Let me know if you get this resolved. I'm going to try and apply again for the GitHub Student Pack later today.
